apps/system/app/[subdomain]/(dashboard)/escuela/[slug]/materias/page.tsx

```
@@ -0,0 +1,333 @@
+    creditos: "",
+    activa: true
+  });
+
+  const setItems = useBreadcrumbStore((state) => state.setItems);
+  
+  
+
+  useEffect(() => {
+    if (escuela) {
+      setItems([
+        { label: `${(escuela?.nombre).toUpperCase()}`, href: `/escuela/${escuela.nombre}` },
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)